src/joystick/hidapi/SDL_hidapijoystick.c

```diff
@@ -82,6 +82,9 @@ static SDL_HIDAPI_DeviceDriver *SDL_HIDAPI_drivers[] = {
     &SDL_HIDAPI_DriverXbox360,
     &SDL_HIDAPI_DriverXbox360W,
 #endif
+#ifdef SDL_JOYSTICK_HIDAPI_XBOXONE
+    &SDL_HIDAPI_DriverGip,
+#endif
```

Comment:
You can just add this to the ifdef below

Reply:
It appears to actually be the wrong ifdef--it should be GIP, not XBOXONE. I'll fix it on Monday.